lib/twitter.js

```diff
@@ -161,7 +161,7 @@ Twitter.prototype.post = function(url, content, content_type, callback) {
     }
     else {
       try {
-        var json = JSON.parse(data || {});
+        var json = data ? JSON.parse(data) : data;
```

Comment:
From the old code I guess the intention of this was to return `{}` if data is null... isn't it?
However with the new code... aren't we returning just `null`?

I'm saying this just in case this change could break anything, although it's clear that as it is right now anything relying on this will be broken already.